docs/TablePartitionsStorageGroups.md

```diff
@@ -0,0 +1,421 @@
+# Table Partitions and Storage Groups
+
+<details>
+<summary>Table of Contents</summary>
+<!-- TOC -->
+
+- [Table Partitions and Storage Groups](#table-partitions-and-storage-groups)
+    - [Introduction](#introduction)
+    - [Deployment Overview](#deployment-overview)
+        - [Prerequisites](#prerequisites)
+        - [Example](#example)
+    - [Managing Storage Groups](#managing-storage-groups)
+        - [Adding Storage Group](#adding-storage-group)
+        - [Removing Storage Group](#removing-storage-group)
+        - [Storage Group Maintenance](#storage-group-maintenance)
+        - [Start Unavailable Storage Group](#start-unavailable-storage-group)
+        - [Storage Group Name Reconfiguration](#storage-group-name-reconfiguration)
+    - [Backup Database with TPSG](#backup-database-with-tpsg)
+        - [Domain Process Labels](#domain-process-labels)
+    - [Restore Database with TPSG](#restore-database-with-tpsg)
+        - [Automatic archive initial import](#automatic-archive-initial-import)
+    - [Troubleshooting](#troubleshooting)
+        - [Storage Group Assignment Mismatch](#storage-group-assignment-mismatch)
+        - [Backup Failure](#backup-failure)
+        - [Storage Group Startup Failure](#storage-group-startup-failure)
+
+<!-- /TOC -->
+</details>
+
+## Introduction
+
+Table partitioning is the mechanism by which a version of a database table row is stored on a subset of Storage Managers (SMs) running in a NuoDB database.
+A table partition is an attribute that can be associated with a database table.
+A storage group represents a logical storage unit that can be serviced by multiple SMs in a database.
+Each partition within a table has a uniquely identified name that maps to one storage group (SG) which represents a logical storage unit that can be serviced by multiple SMs in a database.
+
+This document explains how to configure Table Partitions and Storage Groups (TPSG) in NuoDB Kubernetes deployments.
+
+>**Note**
+> For non-Kubernetes deployments, see  [Table Partitions and Storage Groups][2]. This page expands on the product documentation and is specific to this Helm Chart repository.
+
+## Deployment Overview
+
+NuoDB [database](../stable/database) Helm chart can be installed multiple times in a single Kubernetes namespace where there is one _primary_ and one or more _secondary_ Helm release.
+A storage group is deployed as a separate _secondary_ Helm release which also deploys the SMs servicing this storage group.
+This allows the user to remove any storage group without removing resources installed by the _primary_ database Helm release.
+To simplify TPSG operations and observability, NuoDB Helm charts support one storage group per Storage Manager.
```

Comment:
Should this say "Storage Manager StatefulSet"?

Reply:
Both are correct but adding "StatefulSet" makes in more precise.